apps/examples/mediaplayer/mediaplayer_main.cpp

```diff
@@ -321,22 +316,25 @@ class MediaPlayerController
 	void start()
 	{
 		while (true) {
-			auto test = selectSource();
-			if (test < 0) {
+			vector<string> sourceList = {"Exit APP", "Test PCM", "Test BUFFER", "Test HTTP"};
+			listDirEntries("/rom", sourceList);
```

Comment:
And how about defining "/rom" as a constant variable or macro..?

Reply:
The "/rom" is used only once. Is there any advantage if we use constant variable?

Reply:
/rom is used partially in line 54, (static char TEST_FILE_PATH[128] = "/rom/44100.pcm";).
But replacing it is optional and up to you.